src/gobupload/relate/update.py

```diff
@@ -1003,6 +1065,21 @@ def _is_initial_load(self):
             return True
         return False
 
+    def _force_full_relate(self) -> bool:
+        """Returns True if should force full relate. Full relate is forced when there are many updated src/dst objects
+        to consider.
+
+        :return:
+        """
+        total_src_objects = self._get_count_for(self.src_table_name)
+        total_dst_objects = self._get_count_for(self.dst_table_name)
+
+        changed_src_objects = self._get_count_for(f'({self._src_entities()})')
+        changed_dst_objects = self._get_count_for(f'({self._dst_entities()})')
+
+        return (changed_dst_objects / total_dst_objects) + \
+               (changed_src_objects / total_src_objects) >= _FORCE_FULL_RELATE_THRESHOLD
+
```

Comment:
Ik zou hier de waarde afdrukken zodat in de logging kan worden gevolgd welk pad is gekozen,
iets zoals: changed_dst_objects: <x>, changed_src_objects: <y>, factor <factor>  => <partial|full> relate

Reply:
Ja, je hebt gelijk, dat moest er wel even bij :)